src/core/ctx.ts

```diff
@@ -36,7 +36,9 @@ export function createContext(options: Options = {}, root = process.cwd()) {
 
   const resolvers = options.resolvers ? [options.resolvers].flat(2) : []
 
-  const cache = isCache === false ? false : resolve(root, isCache === true ? 'auto-imports-cache.json' : isCache)
+  const cachePath = isCache === false
+    ? false
+    : resolve(root, typeof isCache === 'string' ? 'string' : 'node_modules/.cache/unplugin-auto-import.json')
```

Comment:
resolve(root, typeof isCache === 'string' ? isCache  : 'node_modules/.cache/unplugin-auto-import.json')

Reply:
i also have another question.

 if default cache file in node_modules: 'node_modules/.cache/unplugin-auto-import.json', 
 people can not share cacheJson, so we can not create dts by cache file.

Reply:
i think this code is right, how would you think about it.

```js
const cachePath = isCache === false
    ? false
    : resolve(root, typeof isCache === 'string' ? isCache : 'auto-imports-cache.json')
```

Reply:
Yeah good catch, PR welcome

Reply:
> if default cache file in node_modules: 'node_modules/.cache/unplugin-auto-import.json',
people can not share cacheJson, so we can not create dts by cache file.

Good point. But I think users should also be able to choice if they want to commit the file to the source control as well. Since it's configurable, I think it's fine.

Reply:
ok, i got it